packages/core/src/components/Tabs/Tab.js

```diff
@@ -1,31 +1,34 @@
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+// @flow
+
+import * as React from 'react';
 import cx from 'classnames';
 
 import css from './Tabs.css';
 
-export default class Tab extends Component {
-  static propTypes = {
-    value: PropTypes.number,
-    selected: PropTypes.bool,
-    className: PropTypes.string,
-    children: PropTypes.node,
-    onClick: PropTypes.func,
-    onFocus: PropTypes.func,
-    onBlur: PropTypes.func,
-  };
+type Props = {
+  value: number,
+  selected: boolean,
+  className: string,
+  children: React.Node,
+  onClick: Function,
+  onFocus: Function,
+  onBlur: Function,
+}
+
+export default class Tab extends React.Component<Props> {
+  component: any;
```

Comment:
we should avoid using `any` as much as possible and use specific types, can use `HTMLButtonElement`